library/src/main/java/com/liulishuo/filedownloader/model/ServiceStatusModel.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2015 LingoChamp Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.liulishuo.filedownloader.model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class ServiceStatusModel implements Parcelable {
+
+    private boolean isHandled;
```

Comment:
I think `isHandled` is useless, if we bind on the several clients, we also can't know the first one is the right one, and bind on the several clients also not support and not be allowed on FileDownloader.